propulsion-hotel/Domain/Config.fs

```diff
@@ -0,0 +1,49 @@
+module Domain.Config
+
+[<RequireQualifiedAccess; NoComparison; NoEquality>]
+type Store =
+    | Memory of Equinox.MemoryStore.VolatileStore<struct (int * System.ReadOnlyMemory<byte>)>
```

Comment:
```suggestion
type Store<'a> =
    | Memory of Equinox.MemoryStore.VolatileStore<'a>
```

unless there's a reason to hardcode the format

Reply:
there tends to be some redundant type args that have to propagate - normally it makes sense to align the type with what your primary concrete store does (which is this tuple if Dynamo, JsonElement for Cosmos, and the ROM for esdb, sss, mdb)

Will try though - thanks for pointing it out because it is def ugly and debatable

Reply:
Turns out doing that results in `obj` being inferred in parts of the code where you don't specify it.
That can be resolved by making it explicit, but that leaks pointlesslely into multiple places (where it's not relevant)